suite-native/notifications/src/components/AnnouncementNotification.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { ReactNode, useState, useEffect } from 'react';
+import Animated, {
+    SlideInUp,
+    SlideOutUp,
+    useAnimatedGestureHandler,
+    useSharedValue,
+    useAnimatedStyle,
+    withTiming,
+    runOnJS,
+} from 'react-native-reanimated';
+import { PanGestureHandler, PanGestureHandlerGestureEvent } from 'react-native-gesture-handler';
+import { TouchableWithoutFeedback } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Box, Text } from '@suite-native/atoms';
+import { InvertedThemeProvider } from '@suite-native/theme';
+
+type AnnouncementNotificationProps = {
+    iconLeft?: ReactNode;
+    iconRight?: ReactNode;
+    title: string;
+    description: ReactNode;
+    onPress: () => void;
+    isHiddenAutomatically?: boolean;
+};
+
+const DISMISS_THRESHOLD = -25;
+
+const ENTER_ANIMATION_DURATION = 200;
+const EXIT_ANIMATION_DURATION = 1000;
+const NOTIFICATION_VISIBLE_DURATION = 5000 + ENTER_ANIMATION_DURATION + EXIT_ANIMATION_DURATION;
+
+const notificationContainerStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingVertical: utils.spacings.small / 2,
+    borderRadius: utils.borders.radii.round,
+    backgroundColor: utils.colors.gray1000,
+    paddingHorizontal: utils.spacings.small / 2,
+    ...utils.boxShadows.small,
+}));
+
+export const AnnouncementNotification = ({
```

Comment:
I am not sure about name, why `Annoucement`? I would prefer something like just `Notification` or `ToastNotification`

Reply:
The naming was based on the design Figma file. But I agree it's not a very accurate name. Renamed to `Notification` in 2f14645f6.